src/main/java/utils/LogFile.java

```diff
@@ -0,0 +1,112 @@
+package utils;
+import java.io.File;
+import java.io.FileWriter;
+import java.util.ArrayList;
+import Tasks.Todo;
+import Tasks.Task;
+import Tasks.Deadline;
+import Tasks.Event;
+
+
+public class LogFile {
+    public static final String LOG_FILE_NAME = "./data/Cubone.txt";
+
+    public static File getLogFile() {
+        File file = new File(LOG_FILE_NAME);
+        // check if the file exists
+        if (!file.exists()) {
+            try {
+                // check if the dictionary exists
+                File directory = new File(file.getParent());
+                if (!directory.exists()) {
+                    directory.mkdirs();
+                }
+                file.createNewFile();
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
+        return file;
```

Comment:
Maybe you could reduce the nesting and make the happy path a bit more prominent?
```
if (file.exists()) {
  return file;
}
... // if file does not exist
```